java/server/test/org/openqa/selenium/grid/sessionqueue/NewSessionQueuerTest.java

```diff
@@ -206,13 +222,18 @@ public void shouldBeClearQueueRemotely() {
 
   @Test
   public void shouldBeClearQueueAndFireRejectedEvent() {
+    AtomicBoolean result = new AtomicBoolean(false);
+
     RequestId requestId = new RequestId(UUID.randomUUID());
-    bus.addListener(NewSessionRejectedEvent.listener(response -> assertEquals(response.getRequestId(), requestId)));
+    bus.addListener(NewSessionRejectedEvent.listener(response ->
```

Comment:
Style nit: Put `response` on the newline, and then the whole lambda can be read more easily.